Emby.Server.Implementations/ApplicationHost.cs

```diff
@@ -114,31 +114,14 @@
 using System.Diagnostics;
 using ServiceStack.Text.Jsv;
 using Microsoft.Extensions.Logging;
+using OperatingSystem = MediaBrowser.Model.System.OperatingSystem;
 
 namespace Emby.Server.Implementations
 {
-    /// <summary>
```

Comment:
I take it these were some sort of IDE comment system? I like the idea of leaving comments as to what various things do for future developers, but the formatting of these is really ugly.

Reply:
I only targeted the comments had no real purpose and don't give a person or an IDE any more useful information. I agree that we should have some comments, though I prefer the idea of putting those in the interfaces (if we had any).

I had a very brief discussion in the jellyfin-dev riot chat with a couple of devs last night and they agreed that there was no point in having these useless comments.